imports/importers.py

```diff
@@ -0,0 +1,73 @@
+import json
+import zipfile
+
+import shapefile
+from django.conf import settings
+from django.contrib.gis.geos import GEOSGeometry
+
+from nature.models import Feature
+
+
+class ShapefileImporter:
+    """A class that allow importing shapefile features to Feature model
+
+    The class makes a few assumptions on the input shapefiles:
+
+    - The shapefiles must be valid, i.e.
+        - all mandatory files (.shp, .shx and .dbf) are present
+        - all files have same name prefix
+        - the number of shapes and records must match
+
+    - The shapefiles must not include additional fields besides the ones
+    listed below and the names must match exactly. Basically they have
+    same names as in database but shapefile has a 10 characters limit
+    on field names.
+
+    - The shapefile may omit some of the fields that are not required by
+    the Feature model.
+    """
+    required_extensions = ('.shp', '.shx', '.dbf')
+    field_mapping = {
```

Comment:
Just a though for the future, nothing to do with yout code:
What if this kind of information would exist for all Models? Or if there would be a function that kind extract this information from the field data in a model. Might be useful in the future, especially if it can be made generic so that there would not have to be a lot of manual coding. Just a though :)

Reply:
I generally follow the the rule of three (three strikes and refactor) on this, so if we need to do the mapping somewhere else again, then we can refactor this.